entries/w/woltlab.com.json

```diff
@@ -1,15 +1,16 @@
 {
   "WoltLab": {
     "domain": "woltlab.com",
+    "tfa": [
+      "totp",
+      "u2f",
+      "email"
+    ],
+    "documentation": "https://www.woltlab.com/account-security/",
     "url": "https://www.woltlab.com/",
```

Comment:
We direct to `https://{domain}/` by default and your website seem to handle that fine :wink: 
```suggestion
```

Reply:
Yes, it does. I'll make the necessary changes in bulk once the other two discussions are resolved.

Reply:
Removed now.